### PR TITLE
Add Game Wireframe

### PR DESCRIPTION
**Pull request by lmworster**
Issue #2

### CODE REVIEW

File: minesweepr.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  <style>
+  table, td {
+    background: #DCDCDC;
+    border: 1px solid #808080;
+    border-collapse: collapse;
+    text-align: center;
+  }
+
+  td {
+    height: 20px;
```
Comment (cparo) [inline minesweepr.html:14]: Is 20px based on other versions of the game? Or is this just what "looks right" when looking at the board?

Comment (lmworster) [reply minesweepr.html:14]: It's just what "looked right."

Comment (cparo) [reply minesweepr.html:14]: Ah. Cool.

File: minesweepr.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    width: 20px;
+  }
+  </style>
+</head>
+
+<body>
+  <table>
```
Comment (cparo) [inline minesweepr.html:21]: For this table, is the idea that later it'll be generated with some JS for a configurable grid size?

Comment (lmworster) [reply minesweepr.html:21]: Yes, that's the idea. A table (as opposed to using divs or another block element) will maintain its row length even if the window is too small to display all of it.

**Review by cparo — APPROVED** [review]
LGTM

**Pull request merged**